pom.xml

```diff
@@ -24,7 +24,7 @@
   </parent>
   <groupId>org.apache.accumulo</groupId>
   <artifactId>accumulo-examples</artifactId>
-  <version>2.0.0-SNAPSHOT</version>
+  <version>2.0.0</version>
```

Comment:
What's the reason for this for this change?
I think this is possibly misleading... because we have not "released" a version with that name.